.github/workflows/codeql.yml

```
@@ -13,9 +13,9 @@ name: "CodeQL Advanced"
 
 on:
   push:
-    branches: [ "develop", "gh-pages" ]
+    branches: [ "main", "gh-pages" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)